### ISSUE TITLE
Parallel Dimension Stacking (Now feature request for dimension height specification for connect_floor command) [Enhancement]

### ISSUE THREAD

**Issue by FloridaMen** [qouteall/ImmersivePortalsMod#1017]
In short, the current functioning of dimension stacking only allows for one stack. Let's say I have an extraneous dimension from another mod called "dimension B." Lets say that I want to put this dimension B under the end, such that when I fall from the end island I end up in dimension B. Perfectly doable. Now let's couple the nether to the overworld, so that by mining through y=0 in the overworld drops me in the nether. The issue arises when you wish to do both. Either dimension B will be above the overworld, or the nether will be above the end (this is especially an issue if you replace the bedrock of the nether with a mine-able block).

**Comment by FloridaMen** [comment]
Just found a second post that speaks of the same issue and you said it was possible to do using just commands. Sorry for the bother.

**Comment by FloridaMen** [comment]
After looking into it, using commands is usually sufficient for what I want to do. However, when playing with mods or datapacks that alter existing dimensions heights (eg. amplified nether) it's not possible to specify at what y value the portal should form, unlike the gui. This would be avoidable in most situations by using the gui for the broken dimension then using the commands for the others, however there are several situations where this would be useful (eg. making a modpack, as I was doing when I found the need for this feature).

**Comment by qouteall** [comment]
Another solution: use alternate5 dimension in the middle, create the dimension stack, and then use command `/portal global delete_global_portal` to remove the unneeded portals to the middle dimension

**Comment by FloridaMen** [comment]
wow that's... kinda genius. thanks for putting so much time into one of, if not the most polished minecraft mod i've ever seen!

**Comment by FloridaMen** [comment]
I've been trying this as per your suggestion, but I'm not quite sure that works.... Even if it did, by my understanding the "/portal global delete_global_portal" command requires a player entity, making it cumbersome for automatic world generation. This can be circumvented by teleporting the player so it isn't a huge deal but either way I can't quite get it to work as described...

**Comment by qouteall** [comment]
Oh I forgot that there are commands to remove global portal without player.
`/portal global connection_floor_remove <dim>`

**Comment by FloridaMen** [comment]
I found a "fix"! I never got this to work:

> Another solution: use alternate5 dimension in the middle, create the dimension stack, and then use command `/portal global delete_global_portal` to remove the unneeded portals to the middle dimension

But, I did make a datapack that accomplishes almost this.
It sacrifices one block off the top of the nether (or respective dimension), and unfortunately allows mobs to spawn on the roof, but it does work. The reason it's not at y=255 instead of y=254 is because the portal gets a bit weird when it's at the height limit (namely not allowing the player through, citing "loading" as an excuse, even though the other side is visible).

Here's the datapack attached for anyone having the same issue, but be advised it is terrible: https://github.com/FloridaMen/dimensionStacker
